app/sidecar/base.toml

```diff
@@ -78,10 +78,10 @@ task-slots = ["sys"]
 
 [tasks.net]
 name = "task-net"
-stacksize = 8000
+stacksize = 10000
 priority = 5
 features = ["mgmt", "h753", "sidecar", "vlan", "vpd-mac", "use-spi-core", "spi3"]
-max-sizes = {flash = 131072, ram = 65536, sram1 = 16384}
+max-sizes = {flash = 131072, ram = 131072, sram1 = 16384}
```

Comment:
I assume this is a result of ~doubling the number of netstack instances required?

Reply:
Yup!  RAM doesn't quite double – it goes from 79420 to 98304 – but we're limited to power-of-two sizes for the RAM region (only flash gets to use spare regions, since it's a tighter commodity).